src/flow/utils.js

```diff
@@ -117,15 +139,26 @@ export const transform = ({ job_parameters, dag_tasks = [] }) => {
   }
 
   steps.forEach((step, i) => {
-    const props = stepProps[step.name]
-    const id = `${step.i || (i + 1)}.${step.name}`
+    const stepName = (step.name).startsWith('bell')
+      ? (step.name).substring(5)
+      : step.name
+
+    const parsedName = stepName
+      .split('_')
+      .join(' ')
+      .replace(/\b\w/g, (l) => l.toUpperCase())
+    const props = stepProps[stepName] || { name: parsedName, level: i }
```

Comment:
because we are still mapping some config to that object, I need to keep the underscores first. If not there, then I added a very similar string parsing as suggested.